iphone/Classes/TiUITabGroup.m

```diff
@@ -620,6 +608,24 @@ - (void)close:(id)args
   RELEASE_TO_NIL(controller);
 }
 
+// This is the focus event we fire on initial open, or when the TabGroup gets focus
+// Note that handleDidShowTab builds the more complex event for switching tabs
+// (and also fires a blur event)
+- (NSDictionary *)focusEvent
```

Comment:
Can you add  - (NSDictionary *)focusEvent  in TiUITabGroup.h file to avoid compiler warning.